assets/api-swagger/swagger.json

```diff
@@ -194,6 +194,66 @@
               "type": "string",
               "default": "0"
             }
+          },
+          {
+            "name": "updateBefore",
+            "in": "query",
+            "description": "Find all orders updated before a certain date.",
+            "required": false,
+            "schema": {
+              "type": "date",
```

Comment:
I don't think the type is right for these, I don't see a 'date' type in the spec (https://swagger.io/specification/). 

I believe it should be a string with the format `date-time`, [Like this for example](https://github.com/Ometria/ometria.docs.assets.external/blob/c98eff1351685fd8c6cc1a941c62988ff09c8e1c/assets/api-swagger/swagger.json#L3098)

Reply:
Updated to date-time

Reply:
It should be like:
```
"type": "string",
"format": "date-time"
```

E.g.
https://github.com/Ometria/ometria.docs.assets.external/blob/0da8717c9a0ab7a2363048ae9648d315d50641b3/assets/api-swagger/swagger.json#L1437

Reply:
Change in https://github.com/Ometria/ometria.docs.assets.external/pull/14/commits/f110739147c7e9c8f0b5e5fcf2b8e7ec3b054d42